### ISSUE TITLE
1.4.0 VAB exception

### ISSUE THREAD

**Issue by Gordon-Dry** [KSPModdingLibs/KSPCommunityFixes#17]
One time in the log, right-clicking parts does not show PAW anymore.
```
NullReferenceException: Object reference not set to an instance of an object
  at KSPCommunityFixes.UIPartActionNumericFloatEdit.Setup (UIPartActionWindow window, Part part, PartModule partModule, UI_Scene scene, UI_Control control, BaseField field) [0x0001c] in <8f81c7b736764f37a0edca532f4ee0dc>:0 
  at UIPartActionWindow.AddFieldControl (BaseField field, Part part, PartModule module) [0x00073] in <cd473063d3a2482f8d93d388d0c95035>:0 
  at UIPartActionWindow.CreatePartList (System.Boolean clearFirst) [0x002bc] in <cd473063d3a2482f8d93d388d0c95035>:0 
  at UIPartActionWindow.Setup (Part part, UIPartActionWindow+DisplayType type, UI_Scene scene) [0x0006a] in <cd473063d3a2482f8d93d388d0c95035>:0 
  at UIPartActionController.CreatePartUI (Part part, UIPartActionWindow+DisplayType type, UI_Scene scene) [0x000f7] in <cd473063d3a2482f8d93d388d0c95035>:0 
  at UIPartActionController.SelectPart (Part part, System.Boolean allowMultiple, System.Boolean overrideSymmetry) [0x001a7] in <cd473063d3a2482f8d93d388d0c95035>:0 
  at UIPartActionController.HandleMouseClick (UnityEngine.Camera cam, System.Boolean allowMultiple) [0x0003f] in <cd473063d3a2482f8d93d388d0c95035>:0 
  at UIPartActionController+<MouseClickCoroutine>d__39.MoveNext () [0x00082] in <cd473063d3a2482f8d93d388d0c95035>:0 
  at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00026] in <12e76cd50cc64cf19e759e981cb725af>:0 
UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```
Leaving VAB and reentering fixed the issue, but... it was reproducable with that part, right-clicking it:
SmartParts/Parts/Smart-Controller/km_smart_srb.cfg

Log and stuff:
[KSP logs 2021-11-20-01.zip](https://github.com/KSPModdingLibs/KSPCommunityFixes/files/7573431/KSP.logs.2021-11-20-01.zip)

**Comment by Gordon-Dry** [comment]
I guess the latest feature UIFloatEditNumericInput is the culprit - although I never used it.
Before I upgraded to 1.4.0 I had no issues with SmartParts or that specific part.
I will turn off that feature now and reboot KSP.

**Comment by Gordon-Dry** [comment]
That one worked. I forgot to mention that after my last session last night.

**Comment by gotmachine** [comment]
Should be fixed in 1.4.1
Thanks for reporting.
